.github/workflows/integration_tests.yml

```diff
@@ -35,10 +35,45 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: rucio/containers
+          fetch-depth: 0
       - uses: actions/checkout@v2
         name: Checkout rucio source
         with:
           path: dev/rucio
+          fetch-depth: 0
+      - name: Select tag for rucio containers
+        shell: bash
+        run: | 
+          # Change to cloned rucio/rucio repo
+          cd $GITHUB_WORKSPACE/dev/rucio
+          
+          # Get current branch and corresponding latest tag in rucio/rucio repo
+          BRANCH=$(git rev-parse --abbrev-ref HEAD)
+          OWNER="${{ github.repository_owner }}"
+          
+          if [ $OWNER != 'rucio' ]; then
```

Comment:
Added a check for `OWNER`, so that integration test runs in forked repositories can still use the correct `rucio/containers` tags. Forked repositories do not contain all the branches and hence the commit history available on `rucio/rucio`, therefore the selected `rucio/containers` can be from a different release line. For example, I did not have 1.26.x tags/branches in my fork, so the containers were picked up from the 1.23.x release line as that existed in my fork.

Reply:
Another note to keep in mind, in case we run into any issues in the future:
The current codebase will go through the commit history shown by the following command:

```
git log --oneline --decorate=short
```

The first tag it finds in the commit history (now->past commits), is used to fetch the correct dev container for integration tests.